app/Models/Role.ts

```diff
@@ -1,22 +1,20 @@
-// import { DateTime } from 'luxon'
-// import { BaseModel, column, ManyToMany, manyToMany } from '@ioc:Adonis/Lucid/Orm'
-// import User from './User'
+import { DateTime } from 'luxon'
+import { BaseModel, column, ManyToMany, manyToMany } from '@ioc:Adonis/Lucid/Orm'
+import User from './User'
 
-// export default class Role extends BaseModel {
-//   @manyToMany(() => User, {
-//     pivotTable: 'role_user',
-//   })
-//   public users: ManyToMany<typeof User>
+export default class Role extends BaseModel {
+  // @manyToMany(() => User)
+  // public users: ManyToMany<typeof User>
 
-//   @column({ isPrimary: true })
-//   public id: number
+  @column({ isPrimary: true, serializeAs: null })
+  public id: number
```

Comment:
why do we need to serialize it as null?

we will never send the role object in the api response, but we can fetch and use it inside the service